### DISCUSSION TITLE
Hi, cannot connect the scene via remote Ominverse Streaming Client.

### DISCUSSION THREAD

**Discussion by Yonggie** [StanfordVL/BEHAVIOR-1K#1013] [Q&A] [answered]
**Describe the bug**
I'm not sure this is with Issac sim or Omnigibson, but I cannot connect to the simulation app on ``server A`` in ``pc B``, which are in the same LAN.

``server A`` actually detected the connection request and shows these:
```
[INFO] [omnigibson.simulator] ---------- Welcome to OmniGibson! ----------
2024-10-23 03:26:01 [30,688ms] [Warning] [omni.syntheticdata.plugin] SdRenderVarPtr missing valid input renderVar LdrColorSDhost
[INFO] [omnigibson.simulator] Imported scene 0.
/home/abrain/anaconda3/envs/omnigibson/lib/python3.10/site-packages/gymnasium/spaces/box.py:418: UserWarning: WARN: Casting input x to numpy array.
  gym.logger.warn("Casting input x to numpy array.")
main: thread_init: already added for thread
main: thread_init: already added for thread
main: thread_init: already added for thread
2024-10-23 03:26:26 [56,141ms] [Warning] [carb.livestream.plugin] onClientEventRaised: NVST_CCE_DISCONNECTED when m_connectionCount 0 != 1
main: thread_init: already added for thread
main: thread_init: already added for thread
main: thread_init: already added for thread
2024-10-23 03:26:30 [59,766ms] [Warning] [carb.livestream.plugin] onClientEventRaised: NVST_CCE_DISCONNECTED when m_connectionCount 4294967295 != 1
main: thread_init: already added for thread
main: thread_init: already added for thread
main: thread_init: already added for thread
2024-10-23 03:26:41 [71,509ms] [Warning] [carb.livestream.plugin] onClientEventRaised: NVST_CCE_DISCONNECTED when m_connectionCount 4294967294 != 1
```

Each time I clicked connect button from ``pc B``, ``sever A`` detected it and the console will show the log below (I tried 3 times at this case):
```
2024-10-23 03:26:26 [56,141ms] [Warning] [carb.livestream.plugin] onClientEventRaised: NVST_CCE_DISCONNECTED when m_connectionCount xxx != 1
main: thread_init: already added for thread
main: thread_init: already added for thread
main: thread_init: already added for thread
```

I noticed that the ``m_connectionCount 4294967295 ``  **increased dramatically fast** and soon it'll crash because of ``segmentation fault``.

**To Reproduce**
1. Just export ``export OMNIGIBSON_HEADLESS=1 `` and use the example scene ``python -m omnigibson.examples.scenes.scene_selector``
2. try to connect the IP via omniverse streaming client provided by the log
3. it'll happen

**Expected behavior**

**Screenshots**

**Desktop (please complete the following information):**
 - OS:  Ubuntu 22.04
 - Isaac Sim Version installed along with omnigibson by ``python -m omnigibson.install``
 - OmniGibson Version [4.1.0]

**Comment by Yonggie** [comment]
It's funny I tried hundreds of times and it finally worked, but I didn't change much. And it again has similar log ``main: thread_init: already added for thread``  but succeed to be connected:
```
main: thread_init: already added for thread
main: thread_init: already added for thread
# and then it worked from now
2024-10-23 06:56:22 [279,806ms] [Warning] [carb] Failed to startup plugin carb.windowing-glfw.plugin (interfaces: [carb::windowing::IGLContext v1.0],[carb::windowing::IWindowing v1.4]) (impl: carb.windowing-glfw.plugin)
...
```

You may close this issue but I just post it so that everybody can see it.

**Comment by hang-yin** [comment]
Hi @Yonggie, sorry for not getting back to you earlier. @cgokmen knows a lot more about streaming though, he can probably help if you run into anything else. Let us know!

**Reply by Gampa-Varun** [reply, marked as answer]
Using these extensions worked
enable_extension("omni.kit.streamsdk.plugins-3.2.1")
enable_extension("omni.kit.livestream.core-3.2.0")
enable_extension("omni.kit.livestream.native")
